docs(admin-tool-faq): create

victoria-miltcheva opened the pull request:
This PR adds documentation for the Admin Tool.

mwhite-ibm reviewed (approved):
This looks good to me! I see there are some minor differences b/w this and the admin tool FAQ on our end, but the differences make sense and I think overall this is great.

victoria-miltcheva commented:
> This looks good to me! I see there are some minor differences b/w this and the admin tool FAQ on our end, but the differences make sense and I think overall this is great.

Yeah I removed references to the exact name of the GH app since it may be different for Kyndryl. Also, I removed some specifics of the leaked token handling process as Kyndryl may have a slightly different one. I'm going to update the Wiki page to link to this one but will keep the IBM-specific things there to avoid confusion.

Also thank you both for reviewing!
